src/resources/js/router.js

```diff
@@ -19,7 +20,14 @@ const router = new VueRouter({
       name: 'Test',
       icon: 'mdi-help-box',
       component: Hello,
-    }],
+    },
+    {
+      path: '/presentationCreation',
```

Comment:
the url should use a `-` synthax and not contain capital letters you should also follow the following rules to create url:
- `<resource>/<id?>/<action>`
in your case for the creation you should have `presentation/creer`. 
And for exemple if you need to do an edit it should look like `presentation/2/modifier`

Reply:
I changed the path name in [1a8be8](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/129/commits/1a8be8c9b35afb28b3822f0aaa9b77c82049639e) but in commented for avoiding #128